### ISSUE TITLE
Website: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.js-playgroundShareEl) Tags: Accessibility, WCAG 1.4.3, color-contrast

### ISSUE THREAD

**Issue by A11Ybot-OP** [rsc/go-get-issue-15410#1]
https://go.dev/

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds 

Target application: The Go Programming Language - https://go.dev/

Element path: .js-playgroundShareEl

Snippet: `<button class="Button js-playgroundShareEl" title="Share in Go Playground">Share</button>`

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.48 (foreground color: #007d9c, background color: #f8f8f8, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1

Environment: Chrome version 102.0.0.0

====
There are 28 other WCAG 1.4.3  issues on the site

**Comment by A11Ybot-OP** [comment]
Title: WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (a[href$="google/"])
Tags: Accessibility, WCAG 2.4.4, WCAG 4.1.2, link-name

Issue: Ensures links have discernible text (link-name - https://accessibilityinsights.io/info-examples/web/link-name)

Target application: The Go Programming Language - https://go.dev/

Element path: a[href$="google/"]

Snippet: <a href="/solutions/google/" class="WhoUsesCaseStudyList-caseStudyLink"> <img loading="lazy" height="48" width="30%" src="/images/logos/google.svg" class="WhoUsesCaseStudyList-logo" alt=""> <p>View case study</p> </a>

How to fix: 
Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Environment: Chrome version 102.0.0.0

====

There are 17 other [WCAG 2.4.4] [WCAG 4.1.2] issues
